README.md

```diff
@@ -6,6 +6,8 @@
 
 ### <em>Community Patterns for the Decentralized Web</em>
 
+We're building a better future for data [on our beta platform](https://archivers.co). come join us!
```

Comment:
Wondering if this should be our tagline below?

Reply:
I'm strongly averse to presenting Data Together as an organization that's creating yet another competing platform in an an increasingly crowded space.  That's not actually what we're doing. Data Together has the capacity to be a rallying point for everyone who thinks we should build the new web around patterns of shared possession and communities coordinating to possess the data they rely on. As part of that, the platform we're building is one example of how to do this but it is not trying to the the ONE solution. It's solving known needs (ie. orgs like EDGI need to give their contributors ways to participate in Harvesting, Storing, Monitoring, etc), and it's addressing them by taking a novel approach to the intersection between decentralized protocol, community organizing, and the role of institutions like libraries. It's a practical solution that we intend to maintain over time, and it's a reference example, but that's in service of the main purpose of Data Together, which is to get everyone strongly aligned around the importance of grappling with the problems of collective action on the decentralized web.  If we don't grapple with those problems, and if the many projects in this space don't align around tackling those problems, the next version of the web will end up being even more prone to monopolies and exploitation than the current HTTP web. Data Together is an opportunity to get ahead of that.

Reply:
Possibly rewrite to:
> We're building a better future for data. The [beta platform at https://archivers.co](https://archivers.co) is one example of where we're going. Come join us!

I will rebase my changes and submit them in a PR after we merge this one, to keep things clean.

Reply:
that would be awesome @flyingzumwalt 🎉

Reply:
Added a commit addressing this @flyinzumwalt.